namui/src/namui/system/file/mod.rs

```diff
@@ -1,6 +1,6 @@
 pub mod bundle;
 mod electron;
 mod init;
+pub(super) use init::init;
 pub mod system_drive;
 pub mod types;
```

Comment:
```suggestion
pub mod system_drive;
pub mod types;
pub(super) use init::init;
```
I think it would be better to separate `mod` and `use`.